templates/microprofile-payara/files/README.md

```diff
@@ -1,14 +1,21 @@
-# Payara Micro Maven template for Platform.sh
+# Payara Micro Maven for Platform.sh
 
-This project provides a starter kit for Payara Micro projects hosted on [Platform.sh](http://platform.sh).
+This project provides a starter kit for Payara Micro projects hosted on Platform.sh.
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+Payara Micro is an open source, lightweight middleware of choice for containerized Java EE (Jakarta EE) microservices deployments.
```

Comment:
Open Source is capitalized.

"middleware of choice" sounds like an endorsement, which we should be avoiding.  Do you mean "middleware-based framework"?